### PR TITLE
[MCC-209996] bump version up to 1.0.0

### PR DESCRIPTION
**Pull request by kenyamat**
@lschreck-mdsol @bvillanueva-mdsol @mdsol/monitoring-sentinels please review. fd52d6c

### CODE REVIEW

**Comment by ghost** [comment]
Looks good - hopefully the next PRs will be easier to check (as no more packages folder in the repository :wink:).

**Comment by cabbott** [comment]
Going to 1.0.0 is great. We'll need a story in next sprint to update documentation, and especially remove the warning in the documentation that this nuget is unstable not meant for production.

File: src/Crichton.Client/Crichton.Client.nuspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>Crichton.Client</id>
     <title>Crichton Hypermedia API Client library</title>
-    <version>0.2.3-alpha</version> <!-- why not just use $version$? Because NuGet 2.8.1 has a bug with PCLs: https://nuget.codeplex.com/workitem/4013 -->
+    <version>1.0.0</version> <!-- why not just use $version$? Because NuGet 2.8.1 has a bug with PCLs: https://nuget.codeplex.com/workitem/4013 -->
     <authors>Ed Andersen</authors>
```
Comment (bvillanueva-mdsol) [inline src/Crichton.Client/Crichton.Client.nuspec]: We can add our team name here

**Comment by kenyamat** [comment]
updated Readme 22e7c1b

**Comment by bvillanueva-mdsol** [comment]
LGTM :+1:

**Comment by ghost** [comment]
I will merge it.

**Comment by edandersen** [comment]
Congrats on version 1.0.0! :-)

**Pull request merged**